### ISSUE TITLE
libs version change

### ISSUE THREAD

**Issue by gspeiliu** [adleywd/WhatsNextSeries#4]
One lib version of the configuration compile changed inconsistently. Could you please tell me why.
compile 'me.grantland:autofittextview:0.2.+'

The version change trace is 0.2.+ -> 0.2.1 -> 0.2.+ -> 0.2.1 

So, could you please tell me why this happens? Thanks

**Comment by adleywd** [comment]
Hello @gspeiliu, I don't actually remember exactly why I made the first change (0.2+ to 0.2.1). Probably for some tests propuse, since in that time I was just developing in beta versions.

The author of the lib suggests to use, in readme:
`dependencies {
    compile 'me.grantland:autofittextview:0.2.+'
}`
But the last change (0.2.+ -> 0.2.1) has a reason.
I started a prod (release) version, so I want the app and their libraries stable as possible.

When you use the plus sign(+) it means that gradle will take the latest version of that lib, for example:
If `me.grantland:autofittextview:0.2.+` and latest version is 0.3.5, then the gradle will compile the lib at the highest version of 0.2.LATEST, so if the lib author made the change 0.2.4 to 0.3.5 the gradle compile at `me.grantland:autofittextview:0.2.4`

If `me.grantland:autofittextview:0.+` and the latest version is 0.3.5, then the gradle will compile the lib at  `me.grantland:autofittextview:0.3.5`

I just found a blog that contains more informations, maybe it explain better: [Don't use dynamic versions for your dependecies](https://blog.danlew.net/2015/09/09/dont-use-dynamic-versions-for-your-dependencies/)

Hope this answer your question 😄
